src/base/datetime/calendar/body/test/accessibility.test.ts

```diff
@@ -0,0 +1,230 @@
+import { expect, fixture, elementUpdated } from "@open-wc/testing";
+import { BaseDateTimeCalendarBody } from "../index";
+
+describe("BaseDateTimeCalendarBody", () => {
+  describe("accessibility", () => {
+    const InitValue = {
+      month: 8,
+      year: 2021,
+      value: "2021-9-22",
+      previousDay: "2021-9-21",
+      previousWeek: "2021-9-15",
+      nextDay: "2021-9-23",
+      nextWeek: "2021-9-29",
+      next2Weeks: "2021-10-6"
+    };
+
+    it('should be highlight previous week when triggered "Up" keyboard event for IE', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(new KeyboardEvent("keydown", { key: "Up" }));
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.previousWeek);
+    });
+
+    it('should be highlight previous week when triggered "ArrowUp" keyboard event', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowUp" })
+      );
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.previousWeek);
+    });
+
+    it('should be highlight previous week when triggered "Down" keyboard event for IE', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
```

Comment:
@nmanhit 
I think don't need to `selectedEl.click()` before dispatch keydown event.
Please try remove it.